module/data/shared/range-field.mjs

```diff
@@ -14,7 +14,7 @@ export default class RangeField extends SchemaField {
   constructor(fields={}, options={}) {
     fields = {
       value: new FormulaField({ deterministic: true }),
-      units: new StringField(),
+      units: new StringField({ required: true, blank: false, initial: "self" }),
```

Comment:
Won't this start throwing errors in `_validateSpecial` if anyone already set `units` to a blank value?

Reply:
Data on disk that already has a blank units will get cleaned to the `initial` value on world load. It would only result in a validation error if there is no valid `initial` to clean to.